### PR TITLE
distro: update flex-bsp overrides

### PR DESCRIPTION
**Pull request by aarslan7**
flex-bsp has moved to machine overrides. Update all instances accordingly

JIRA: SB-21746

### CODE REVIEW

**Comment by aarslan7** [comment]
env details for above changes:
```
# $IMAGE_FEATURES_DEVELOPMENT [2 operations]
#   set? /var/aarslan1/flex-13/build2/conf/local.conf:27
#     "debug-tweaks"
#   :append[flex-bsp] /var/aarslan1/flex-13/build2/conf/local.conf:28
#     " codebench-debug ssh-server-openssh"
# pre-expansion value:
#   "debug-tweaks codebench-debug ssh-server-openssh"
IMAGE_FEATURES_DEVELOPMENT="debug-tweaks codebench-debug ssh-server-openssh"
```
```
# $ALLOWED_TUNING [2 operations]
#   set? /var/aarslan1/flex-13/meta-sokol-flex/meta-sokol-flex-common/classes/required_toolchain_config.bbclass:5
#     "${TUNE_PKGARCH}"
#   override[flex-bsp]:set /var/aarslan1/flex-13/meta-sokol-flex/meta-sokol-flex-distro/conf/distro/sokol-flex.conf:182
#     "armv5te armv7ahf-neon armv7a-neon armv7at armv8a i686 riscv64 x86_64"
# pre-expansion value:
#   "armv5te armv7ahf-neon armv7a-neon armv7at armv8a i686 riscv64 x86_64"
ALLOWED_TUNING="armv5te armv7ahf-neon armv7a-neon armv7at armv8a i686 riscv64 x86_64"
```

**Comment by abelal** [comment]
Related https://github.com/MentorEmbedded/meta-flex-ref-bsps/pull/95 / https://github.com/MentorEmbedded/meta-flex-ref-bsps/pull/96

**Review by abelal — APPROVED** [review]
This is good to go but should only be merged once the parent PRs are merged

**Pull request merged**
